app/code/Magento/CatalogGraphQl/README.md

```diff
@@ -1,4 +1,4 @@
-# CatalogGraphQl
+# QuoteGraphQl
```

Comment:
Looks like accidental change

Reply:
Indeed. Will fix that. Thank you for noticing!